vast/integration/integration.py

```diff
@@ -610,7 +610,7 @@ def match(x, names):
                 test_result = Result.FAILURE
                 for i in range(0, args.repetitions):
                     test_result = tester.run(name, definition)
-                    if test_result is Result.TIMEOUT:
+                    if test_result is not Result.SUCCESS:
```

Comment:
Shouldn't this be `if not a is b`, rather than `if a is not b`? Or does Python just handle this the same way (the precedence is confusing).

Reply:
https://peps.python.org/pep-0008/#programming-recommendations

Reply:
Sorry for the brevity, I was a bit preoccupied.

Reply:
No worries, I think your reply actually had the perfect level of information density.